cmake/required.cmake

```diff
@@ -35,4 +35,5 @@ message(STATUS "[python3] python3 found at: ${PYTHON}")
 message(STATUS "[fpp-tools] fpp-depend found at: ${FPP_DEPEND}")
```

Comment:
We should probably at some point make these debug messages, and just print the FPP tools path .